src/main/java/com/stirante/RuneChanger/client/ChampionSelection.java

```diff
@@ -116,7 +116,7 @@ private void findSelectedChampion(LolChampSelectChampSelectSession session) {
                             //noinspection unchecked
                             Map<String, Object> selectAction = (Map<String, Object>) obj;
                             if (selectAction.get("type").equals("pick") &&
-                                    selectAction.get("actorCellId") == selection.cellId) {
+                                    selectAction.get("actorCellId").equals(selection.cellId)) {
```

Comment:
Have you checked if this works?

Reply:
Not sure if it would but since they are both longs i reverted this change.